src/Integration/Notifications/VisualStudioStatusBarHelper.cs

```diff
@@ -44,10 +44,7 @@ public static void AddStatusBarIcon(FrameworkElement statusBarIcon)
                 return;
             }
 
-            var root = VisualTreeHelper.GetChild(
-                VisualTreeHelper.GetChild(Application.Current.MainWindow, 0), 0);
-
-            var dockPanel = GetStatusBarPanel(root);
+            var dockPanel = FindSccStatusBarParentPanel(Application.Current.MainWindow);
             if (dockPanel == null)
             {
                 return;
```

Comment:
It might be worth a debug fail here. This is why it failed silently in 2019 and no one knew.